driver/src/main/java/org/neo4j/driver/internal/handlers/RunResponseHandler.java

```diff
@@ -22,28 +22,31 @@
 import java.util.concurrent.CompletableFuture;
 
 import org.neo4j.driver.Value;
+import org.neo4j.driver.exceptions.AuthorizationExpiredException;
+import org.neo4j.driver.internal.async.UnmanagedTransaction;
+import org.neo4j.driver.internal.spi.Connection;
 import org.neo4j.driver.internal.spi.ResponseHandler;
 import org.neo4j.driver.internal.util.MetadataExtractor;
 import org.neo4j.driver.internal.util.QueryKeys;
 
 public class RunResponseHandler implements ResponseHandler
 {
-    private final CompletableFuture<Throwable> runCompletedFuture;
+    private final CompletableFuture<Void> runFuture;
```

Comment:
Very nice change… Completable future of throwable was weird.

Reply:
Yeah 👍